transit_odp/templates/api/avl_subscription_detail.html

```diff
@@ -0,0 +1,78 @@
+{% extends "page.html" %}
+{% load static %}
+{% load breadcrumbs %}
+{% load i18n %}
+{% load crispy_forms_tags %}
+{% block title_tag %}
+  {% if form.errors %}
+    {% trans "Error: " %}
+  {% endif %}
+  {% trans "Manage subscription" %}
+{% endblock %}
+{% block breadcrumb.content %}
+  {{ block.super }}
+  {% breadcrumb_url 'API services' 'api-select' host hosts.data %}
+  {% breadcrumb_url 'Location data API Service' 'api:buslocation-api' host hosts.data %}
+  {% breadcrumb_url 'Manage subscriptions' 'api:buslocation-manage-subscriptions' host hosts.data %}
+  {% breadcrumb_url subscription.name %}
+{% endblock %}
+{% block content %}
+  <h1 class="govuk-heading-xl">{{ subscription.name }}</h1>
+  <hr class="govuk-section-break govuk-section-break--l govuk-section-break--invisible" />
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <table class="govuk-table dataset-property-table">
+        <tbody class="govuk-table__body">
+          <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">{% trans "Name" %}</th>
+            <td class="govuk-table__cell dont-break-out" colspan="2">{{ subscription.name }}</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">{% trans "ID" %}</th>
+            <td class="govuk-table__cell dont-break-out" colspan="2">{{ subscription.id }}</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">{% trans "Status" %}</th>
+            <td class="govuk-table__cell dont-break-out" colspan="2">{{ subscription.status }}</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">{% trans "URL" %}</th>
+            <td class="govuk-table__cell dont-break-out" colspan="2">{{ subscription.url }}</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">{% trans "Requestor Ref" %}</th>
```

Comment:
```suggestion
            <th scope="row" class="govuk-table__header">{% trans "Requestor ref" %}</th>
```

should these titles all be lowercase?